test/test_quantization.py

```diff
@@ -259,9 +260,29 @@ def checkQuantized(model):
         checkQuantized(model)
 
         # test one line API
-        model = quantize(ManualQuantModel(), default_eval_fn, calib_data)
+        model = quantize(ManualQuantModel().eval(), default_eval_fn, calib_data)
         checkQuantized(model)
 
+class QuantizationAwareTrainingTest(QuantizationTestCase):
+    def test_manual(self):
+        model = ManualQATModel()
+        model.qconfig = default_qat_qconfig
+
+        prepare(model)
+        self.checkObservers(model)
+
+        default_eval_fn(model, calib_data)
+        convert(model)
+
+        def checkQuantized(model):
+            self.assertEqual(type(model.fc1), nnq.Linear)
+            self.assertEqual(type(model.fc2), nnq.Linear)
+            default_eval_fn(model, calib_data)
+
+        model = ManualQATModel()
+        model.qconfig = default_qat_qconfig
+        model = quantize(model, default_train_fn, train_data)
+        checkQuantized(model)
```

Comment:
Can we also have a test that allows for insertion of fake-quant on a model setup to be in evaluation mode?.

Reply:
added a test, we just need to call prepare